src/actions/userActions.js

```diff
@@ -123,6 +123,29 @@ export function updateUserInfo(userInfo, userId) {
     };
 }
 
+export function uploadProfilePicture(file, userId) {
```

Comment:
might be able  to factor some part of this out since the roaster upload prof pic code is the same except for the url.